### PR TITLE
find source dir for libunwind when building ldd for Clang versions >= 12.0.1

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyblocks/c/clang.py [hunk 1]
```diff
@@ -125,6 +125,7 @@ def extract_step(self):
             libcxx/       Unpack libcxx-*.tar.gz here
             libcxxabi/    Unpack libcxxabi-*.tar.gz here
             lld/          Unpack lld-*.tar.gz here
+        libunwind/        Unpack libunwind-*.tar.gz here
```
[the remark this answers is not in the document]
Comment (jfgrimm) [reply easybuild/easyblocks/c/clang.py:128]: The indentation is correct, it sits at the same directory level as the llvm source

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Just waiting for the build reports to come in.

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _Clang-11.0.1-GCCcore-10.2.0.eb_ 

Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3600.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/81bb42d9f8e8b5c4f447693c10ebcffe for a full test report.

**Review by boegel — APPROVED** [review]

**Pull request merged**